test/testdata/parser/error_recovery/unop_binop.rb.parse-tree.exp

```diff
@@ -0,0 +1,1399 @@
+Class {
```

Comment:
It's very hard for me to eyeball that this 1.4k line file captures the parse result that we want. It would be **much** easier to skim using `desugar-tree` instead of `parse-tree`, which turns it from a 1.4k line file into a very eyeball-able ~100 line file.

Doing this locally is how I found the bug I mention above relating to `&&` and `||`.